portal-api/src/v1/users/controller.js

```diff
@@ -138,35 +139,37 @@ exports.findByEmail = async (email, callback) => {
 
 exports.create = async (user, callback) => {
   const insert = {
-    'user-status': ACTIVE,
-    timezone: user.timezone || 'Europe/London',
+    'user-status': USER.STATUS.ACTIVE,
+    timezone: USER.TIMEZONE.DEFAULT,
     ...user,
   };
 
   delete insert.password;
   delete insert.passwordConfirm;
 
-  const collection = await db.getCollection('users');
-  const createUserResult = await collection.insertOne(insert);
+  if (payloadVerification(insert, PAYLOAD.PORTAL.USER)) {
+    const collection = await db.getCollection('users');
+    const createUserResult = await collection.insertOne(insert);
 
-  const { insertedId: userId } = createUserResult;
+    const { insertedId: userId } = createUserResult;
 
-  if (!ObjectId.isValid(userId)) {
-    throw new Error('Invalid User Id');
-  }
+    if (!ObjectId.isValid(userId)) {
+      throw new Error('Invalid User Id');
+    }
+
+    const createdUser = await collection.findOne({ _id: { $eq: userId } });
 
-  const createdUser = await collection.findOne({ _id: { $eq: userId } });
+    const sanitizedUser = sanitizeUser(createdUser);
 
-  const sanitizedUser = sanitizeUser(createdUser);
+    if (sanitizedUser.username && sanitizedUser.username.includes('@')) {
```

Comment:
This is a tad surprising without the comment. I feel the comment could be something like 
```
// We have test users whose usernames are not email addresses.
// We use the below to avoid sending emails to these users
```

Also, do we have a ticket to fix this? Might be worth referencing if we do

Reply:
Have added the associated ticket in the coment